templates/overrides/semantic/webbasic/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,103 @@
+            <menulink class="blue item" controller-name="Home" action-name="About" menu-text="About"></menulink>
+            <menulink class="blue item" controller-name="Home" action-name="Contact" menu-text="Contact">Contact</menulink>
+            
+        </div>
+    </div>
+
+    <!-- Sidebar Menu -->
+    <div class="ui vertical inverted sidebar menu">
+        <menulink class="blue item" controller-name="Home" action-name="Index" menu-text="Home"></menulink>
+        <menulink class="blue item" controller-name="Home" action-name="About" menu-text="About"></menulink>
+        <menulink class="blue item" controller-name="Home" action-name="Contact" menu-text="Contact"></menulink>
+        
+    </div>
+
+    
+
+<!-- Page Contents -->
+<div class="pusher">
+    <div class="ui container content">
+        @RenderBody()
+    </div>
+    
+    <div class="ui vertical footer segment">
+        <div class="ui container">
+            <div class="ui stackable  divided equal height stackable grid">
+                <div class="three wide column">
+                    <h4 class="ui header">About</h4>
+                    <div class="ui  link list">
+                        <a href="#" class="item">Sitemap</a>
+                        <a href="#" class="item">Contact Us</a>
+                        <a href="#" class="item">Link</a>
+                        <a href="#" class="item">Link</a>
+                    </div>
+                </div>
+                <div class="three wide column">
+                    <h4 class="ui header">Services</h4>
+                    <div class="ui  link list">
+                        <a href="#" class="item">Service One</a>
+                        <a href="#" class="item">Service Two</a>
+                        <a href="#" class="item">Service Three</a>
+                        <a href="#" class="item">Service Four</a>
+                    </div>
+                </div>
+                <div class="ten wide column">
+                    <h4 class="ui header">Footer Header</h4>
+                    <p>Yes I know you probably disregarded the earlier boasts as non-sequitor filler content, but its really true. It took years of gene splicing and combinatory DNA research, but our bananas can really dance.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+    <environment names="Development">
+        <script src="~/lib/jquery/dist/jquery.js"></script>
+        <script src="~/lib/semantic/dist/semantic.js"></script>
+        <script src="~/js/site.js"></script>
+    </environment>
+    <environment names="Staging,Production">
+        <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.4.min.js"
+                asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
+                asp-fallback-test="window.jQuery">
+        </script>
+        <script src="//oss.maxcdn.com/semantic-ui/2.1.8/semantic.min.js"
```

Comment:
It has to be full link with https in templates:
https://github.com/OmniSharp/generator-aspnet/commit/73b82809c1694ad69b66532cb04eac7b1540c865
I know it will hurt developers when working locally, but that's requirement:
https://twitter.com/paul_irish/status/588502455530311680?lang=en